sw/device/lib/dif/dif_plic.c

```diff
@@ -89,39 +99,42 @@ static ptrdiff_t plic_offset_from_reg0(dif_plic_irq_id_t irq) {
  * the bit position within a register for a specifci IRQ source ID (ID 32 would
  * be bit 0).
  */
-static uint8_t plic_reg_bit_index_from_irq_id(dif_plic_irq_id_t irq) {
+static uint8_t pic_irq_bit_index(dif_plic_irq_id_t irq) {
```

Comment:
nit: `plic_...` instead of `pic_...`?

Reply:
I guess this would also apply at the callsites.

Reply:
That's a typo the spell-checker didn't catch. Thanks Alphan!